dev/mm/deploy.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
I'm opting for helmfile over a bash script here.

https://github.com/broadinstitute/datarepo-helm-definitions/blob/master/dev/ms/helmfile.yaml

https://github.com/roboll/helmfile
it will update the repos and deploy all the charts.

Reply:
Added helmfile to dev/mm also. Thanks for the pointer. Also left the bash script there for reference.